etc/galasa-tests/README.md

```diff
@@ -0,0 +1,226 @@
+#### ✅ Example Format
+```properties
+secure.credentials.YOUR_CREDENTIAL_TAG.username=your-username
+secure.credentials.YOUR_CREDENTIAL_TAG.password=your-password
+```
+#### ✅ Sample Property Configuration
+```properties
+# Default CICS logon details
+cicsts.default.logon.gm.text=******
+cicsts.default.logon.initial.text=ENTER
+# Provisioning
+cicsts.provision.type=dse
+# z/OS cluster and image setup
+zos.cluster.DEFAULT.images=YOUR_IMAGE_ID
+zos.dse.tag.PRIMARY.clusterid=YOUR_CLUSTER_ID
+zos.dse.tag.PRIMARY.imageid=YOUR_IMAGE_ID
+zos.dse.tag.RUN.clusterid=YOUR_CLUSTER_ID
+zos.dse.tag.RUN.imageid=YOUR_IMAGE_ID
+# Image credentials and access
+zos.image.YOUR_IMAGE_ID.credentials=YOUR_CREDENTIAL_TAG
+zos.image.YOUR_IMAGE_ID.default.hostname=your.host.name
+zos.image.YOUR_IMAGE_ID.ipv4.hostname=your.host.name
+zos.image.YOUR_IMAGE_ID.javahome=JAVA_HOME
+zos.image.YOUR_IMAGE_ID.liberty.install.dir=LIBERTY_INSTALL_DIR
+zos.image.YOUR_IMAGE_ID.max.slot=HI
+zos.image.YOUR_IMAGE_ID.max.slots=50
+zos.image.YOUR_IMAGE_ID.ports=20{3-9}{0-9}{0-9},21{0-3}{0-9}{0-9},21{0-2}{0-9}
+zos.image.YOUR_IMAGE_ID.sysplex=YOUR_SYS_PLEX
+zos.image.YOUR_IMAGE_ID.telnet.port=TELNET_PORT
+zos.image.YOUR_IMAGE_ID.telnet.tls=false
+# Terminal emulator settings
+zos3270.terminal.output=json,png
+# Console and batch job settings
+zosbatch.batchjob.YOUR_IMAGE_ID.restrict.to.image=false
+zosconsole.console.YOUR_IMAGE_ID.restrict.to.image=false
+# z/OSMF config
+zosmf.image.YOUR_IMAGE_ID.servers=YOUR_SERVER_ID
+zosmf.server.YOUR_SERVER_ID.image=YOUR_IMAGE_ID
+zosmf.server.YOUR_IMAGE_ID.https=true
+zosmf.server.YOUR_IMAGE_ID.port=PORT_NUMBER
+zosmf.sysplex.YOUR_SYS_PLEX.default.servers=YOUR_IMAGE_ID
+# CBSA-specific tags and region info
+cbsa.cbsa.zos.image.tag=PRIMARY
+cbsa.cbsa.region.image.tag=PRIMARY
+cbsa.region.applid=YOUR_CBSA_APPLID
+cbsa.region.credentials.tag=YOUR_CREDENTIAL_TAG
+# WebUI endpoint (React app)
+cbsa.webui.url=http://your-webui-host:PORT/webui-1.0/#/profile/Admin
+# Selenium config (for local or grid execution)
+# For Local
+selenium.local.driver.FIREFOX.path=LOCAL_PATH
+selenium.driver.type=FIREFOX,CHROME etc
+# For Grid / Standalone Deployment
+selenium.driver.type=grid
+selenium.default.driver=grid
+selenium.grid.endpoint=ENDPOINT_URL
+# Galasa OBR/Test Stream config (based on Maven Repo)
+framework.test.stream.main.repo=https://your-repo-host/
+framework.test.stream.main.obr=mvn:com.yourorg/your.obr.artifact/0.01/obr
+framework.test.stream.main.location=https://your-repo-host/path/toyour/testcatalog.json
+```
+---
+## ▶️ Running the Tests
+### Run Tests Locally
+```bash
+./gradlew clean build publishToMavenLocal
+galasactl runs submit local --class com.ibm.cics.cip.cbsa.galasatests.install/com.ibm.cics.cip.cbsa.galasa.tests.install[TestClassName] --obr mvn:com.ibm.cics/com.ibm.cics.cip.cbsa.galasatests.obr/0.0.1/obr --log -
+```
+### Run Tests within Ecosystem
+```bash
+galasactl runs prepare \
+        --stream "main" \
```

Comment:
The `main` test stream would need to be created prior to running this `galasactl runs prepare` command. You can do this by creating a GalasaStream YAML resource, like this:

```yaml
apiVersion: galasa-dev/v1alpha1
kind: GalasaStream
metadata:
    name: main
    description: CBSA Galasa tests
data:
    isEnabled: true
    repository:
        url: https://your-maven-repo/path/to/cbsa/test/material
    obrs:
        - group-id: com.ibm.cics
          artifact-id: com.ibm.cics.cip.cbsa.galasatests.obr
          version: 0.0.1
    testCatalog:
        url: https://your-maven-repo/path/to/testcatalog.json
```

Then applying this to your Galasa Ecosystem by running:

```bash
galasactl resources apply -f <your-galasa-stream-file>.yaml --bootstrap [bootstrap] --log -
```